public/sass/pages/_dashboard.scss

```diff
@@ -102,22 +102,22 @@ div.flot-text {
 
   .fa {
     position: relative;
-    top: -2px;
+    top: -4px;
     left: -5px;
-    font-size: 90%;
+    font-size: 75%;
   }
 
   &--info {
     display: block;
-    background: $blue-dark;
+    background: mix($panel-bg, $body-bg, 35%);
```

Comment:
I went through them all - Im comfortable mixing two existing colors in this case, but not introducing another variable. 

The pure $body-bg was too stark of a contrast, and seemed strange next to the border, and everything else is lighter. Nothing else in the darker spectrum looked good. :/

I understand the concern though - and you're objectively right. It is a new color, but it's derived from our palette, which feels like an important distinction.